docs/other/osg-token-renewer.md

```diff
@@ -0,0 +1,195 @@
+
+To create a new client account named `<ACCOUNT_SHORTNAME>`:
+
+- Create a corresponding file named `/etc/osg/tokens/<ACCOUNT_SHORTNAME>.pw`
+  with the encryption password to use for this client account.
+- Consult the 
+  [Requesting Tokens](https://opensciencegrid.org/technology/software/requesting-tokens/)
+  document to determine which scopes you will need for this client account.
+- Run the setup script as follows:
+  ```console
+  root@server # osg-token-renewer-setup.sh <ACCOUNT_SHORTNAME> <SCOPES...>
+  ```
+  For example,
+  ```console
+  root@server # osg-token-renewer-setup.sh myaccount123 wlcg offline_access
+  ```
+- You will be prompted on the console to visit a WLCG web link to authorize
+  the client request with a passcode printed on the console.
+  Follow the prompts (visit the web link, enter the request passcode,
+  log in with your WLCG INDIGO IAM user account, and authorize the request).
+- If this succeeds you will be prompted with a new
+  `[account <ACCOUNT_SHORTNAME>]` section to add to your `config.ini`.
+  Go ahead and add it, replacing the example section if it's still there.
+
+Next you can configure one or more tokens for this client account.
+
+
+Configuring tokens for an OIDC client account
+---------------------------------------------
+
+After you have created an OIDC client account and added it to
+`/osg/token-renewer/config.ini`, you need to create a corresponding `token`
+section in the config for each token that should be generated for this account
+(possibly with different options).
+
+Choose a `<TOKEN_NAME>` and add a new `[token <TOKEN_NAME>]` section
+(replacing the example section if it's still there).
+
+The `account` option in this section must match the `<ACCOUNT_SHORTNAME>`
+for the corresponding `[account <ACCOUNT_SHORTNAME>]` section.
+
+The `token_path` option specifies the output path where the generated token
+will be saved.
+
+Optionally, you may also specify any of the following options, which will
+be passed to the `oidc-token` command when generating the token:
+
+| Option         | Description                                               |
+|:---------------|:----------------------------------------------------------|
+| `audience`     | list of audiences to pass via `--aud` option              |
+| `scope`        | list of scopes to pass via `--scope` option               |
+| `min_lifetime` | min token lifetime in seconds to pass via `--time` option |
+
+!!! note
+    For tokens used against an HTCondor-CE, set the `audience` option to
+    `<CE FQDN>:<CE PORT>`.
+
+
+Running the token renewal service
+---------------------------------
+
+The OSG token renewal service is set up as a "oneshot" systemd service,
+which runs under the `osg-token-svc` user, sets up an `oidc-agent`,
+adds the relevant OIDC client accounts as specified in the `config.ini`
+with `oidc-add`, and generates the tokens with `oidc-token`.
+
+This service is set to run via a systemd timer nightly at midnight.
```

Comment:
Add tables like here https://opensciencegrid.org/docs/data/stashcache/install-cache/#managing-stashcache-and-associated-services and here https://opensciencegrid.org/docs/data/stashcache/install-cache/#managing-stashcache-and-associated-services, listing `osg-token-renewer.service` and `osg-token-renewer.timer`